### DISCUSSION TITLE
How to remove blank white space in Mantine Carousel?

### DISCUSSION THREAD

**Discussion by realyukii** [mantinedev/mantine#4350] [Q&A]
how to remove the blank white space that shown in this image?
![image](https://github.com/mantinedev/mantine/assets/126035476/63fb77d4-a7a5-4830-a241-0f7799aa9034)

here's [the code](https://github.com/Reyuki-san/bilibili-dl/blob/dev/apps/web/components/Home/Anime.tsx#L18-L65)
and [the live preview](https://bilibili-dl-web.vercel.app/)

**Comment by yverby** [comment]
Try the following
```js
<Carousel containScroll="trimSnaps">...</Carousel>
```

**Reply by realyukii** [reply]
Thanks for the answer, it's work as well! but there are other things I want to achieve
it's possible to put the  control button (an arrow) outside the content? i try use `controlsOffset="xs"` but it's not work like this image
![image](https://github.com/mantinedev/mantine/assets/126035476/026b9c06-360c-45f6-b226-bf8c8416eac0)

the image above show the left arrow outside the content, i try `align={0.1}` but it's not work since i implement `containScroll="trimSnaps"`

**Reply by realyukii** [reply]
I wish I could find any kind of designation to implement this kind of thing:
![image](https://github.com/mantinedev/mantine/assets/126035476/f81edb94-1b72-4602-90be-14324592061e)
